editor/src/clj/editor/material.clj

```diff
@@ -191,6 +219,12 @@
                     (:array-sampler-names augmented-vertex-shader-info)
                     (:array-sampler-names augmented-fragment-shader-info)))
 
+            resource-binding-namespaces
+            (into []
+                  (distinct (concat
+                              (:resource-binding-namespaces augmented-vertex-shader-info)
+                              (:resource-binding-namespaces augmented-fragment-shader-info))))
+
```

Comment:
Why is it a vector? You're both creating it as a set and using it as a set. 😅

Although, the way we're using this, it seems we could turn it into a `Regex` here and simply use `string/replace` in `strip-resource-namespaces`.